camel/types/enums.py

```diff
@@ -374,3 +374,10 @@ def is_openai(self) -> bool:
             VoiceType.NOVA,
             VoiceType.SHIMMER,
         }
+
+
+class ReturnFormat(Enum):
```

Comment:
Make the name more specific, such as
```suggestion
class JinaReturnFormat(Enum):
```